wurst/_handles/Player.wurst

```diff
@@ -29,6 +29,12 @@ public function player.addLumber(int val)
 public function player.addState(playerstate state, int val)
 	SetPlayerState(this, state, this.getState(state) + val)
 
+/** Clears the selection for the player, no matter which
+	unit is selected. */
```

Comment:
Maybe "Clears selection for the player, deselecting all widgets"

It's not only units :)